packages/core/src/node/runtimeModule/prismLanguages.ts

```diff
@@ -5,14 +5,20 @@ import { FactoryContext, RuntimeModuleID } from '.';
 export async function prismLanguageVMPlugin(context: FactoryContext) {
   const { config } = context;
   const { highlightLanguages = [] } = config.markdown || {};
+  const aliases: Record<string, string[]> = {};
   const languageMeta = uniqBy(
     [...DEFAULT_HIGHLIGHT_LANGUAGES, ...highlightLanguages].map(language => {
-      const [alias, name] = Array.isArray(language)
-        ? language
-        : [language, language];
+      const isArray = Array.isArray(language);
+      const [alias, name] = isArray ? language : [language, language];
+
+      if (isArray) {
+        const temp = aliases[name] || (aliases[name] = []);
+        temp.push(alias);
```

Comment:
Users may repeatedly configure existing default values, we may need to determine whether the element already exists in the array, but it will have no impact on subsequent logic.

eg

``` ts
markdown: {
    highlightLanguages: [
      ['js', 'javascript'],
      ['ts', 'typescript'],
    ],
  },
```

result in 

```ts
aliases:  {
  javascript: [ 'js', 'js' ],
  typescript: [ 'ts', 'ts' ],
  tsx: [ 'jsx', 'mdx' ],
  'xml-doc': [ 'xml' ],
  markdown: [ 'md' ]
}
```